bot/resources/easter/save_the_planet.json

```diff
@@ -0,0 +1,17 @@
+{
+    "embeds": [
```

Comment:
If the json is meant to contain the embeds, it should probably just have the format you give to Embed.from_dict already here.

Reply:
So have it like this?
```
[
    {},
    {},
    ...
]

Reply:
> What is your username on discord @ChrisGallardo? 😄

myPetWiiUReggie